string drone_sn
string car_sn
